_docs/tasks/security/authn-policy.md

```diff
@@ -88,6 +86,46 @@ NAME          AGE
 enable-mtls   1m
 ```
 
+Add this destination rule to config client side to use mTLS:
+
+```bash
+cat <<EOF | istioctl create -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "enable-mtls"
+  namespace: "foo"
+spec:
+  host: *.foo.svc.local.cluster
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+```
+
+> Note:
+* This rule is based on the assumption that there is no other destination rule in the system. If it's not the case, you need to modify traffic policy in existing rules accordingly.
+* `*.foo.svc.local.cluster` matches all services in namespace `foo`.
+* You can also use the mode `MUTUAL` and fill in the paths for key and certificates as below. This is mainly to illustrate the settings that Istio uses behind the scene. In production environment, it's recommended to use `ISTIO_MUTUAL` mode.
```

Comment:
Also add a sentence to explain for ISTIO_MUTUAL, people don't need to specify the path and the path are "/etc/certs/cert-chain.pem...". I think explicitly mentioning this is useful to avoid confusion.

Reply:
+1

Also probably we can add link to the `ISTIO_MUTUAL` in api reference (if exists). I don't think we need to mention `MUTUAL` here which just introduce confusion without further explanation for distinguishing.

Reply:
Remove mention of MUTUAL